pandas/core/arrays/categorical.py

```diff
@@ -1081,20 +1081,26 @@ def remove_unused_categories(self, inplace=False):
             return cat
 
     def map(self, mapper):
-        """Apply mapper function to its categories (not codes).
+        """
+        Map categories (not codes) using input correspondence (a dict,
+        Series, or function).
+
+        Maps the categories to new categories. If the mapping
+        correspondence maps each original category to a different new category
+        the result is a Categorical which has the same order property as
+        the original, otherwise an np.ndarray is returned.
+
+        If a dictionary or Series is used any unmapped category is mapped to
```

Comment:
When referencing Python types use the actual name and place in backticks, so \`dict\` here. For pandas types can also link to the class so :class:\`~pandas.Series\`